File: translations/codenet/deepseek_commented/C2J/starcoder/Code/s033933040.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int a, b, mawari, menseki;
		a = sc.nextInt();
		b = sc.nextInt();
		mawari = 2 * (a + b);
		menseki = a * b;
		System.out.println(menseki + " " + mawari);
	}
}

File: translations/codenet/deepseek_commented/C2Go/granite/Code/s033933040.go

package main

import "fmt"

func main() {
	var a, b, mawari, menseki int
	fmt.Scan(&a, &b)
	mawari = 2 * (a + b)
	menseki = a * b
	fmt.Printf("%d %d\n", menseki, mawari)
}
